script/libtest.bash

```diff
@@ -457,6 +457,23 @@ create_http_trigger(){
     fi
     eval $command
 }
+
+create_http_trigger_cors(){
```

Comment:
you don't need to define a different function, you can add a parameter to `create_http_trigger` to add the cors parameter